docs/Custom_Operation_for_GPUs.md

```diff
@@ -1143,12 +989,12 @@ jax.config.update("experimental_xmap_spmd_lowering_manual", True)
 
 def xmap_rms_norm(x, weight, *, device_count):
     reshaped = x.reshape(device_count, x.shape[0] // device_count, *x.shape[1:])
-    xmap_axes = (("a", None, None, None), (None, None))
+    xmap_axes = (("x", None, None, None), (None, None))
     xmapped = xmap(
         rms_norm,
         in_axes=xmap_axes,
```

Comment:
i might just inline in_axes and out_axes

Reply:
Done